modules/default/weather/providers/openmeteo.js

```diff
@@ -401,7 +401,7 @@ WeatherProvider.register("openmeteo", {
 			currentWeather.temperature = parseFloat((weather.apparent_temperature_max + weather.apparent_temperature_min) / 2);
 			currentWeather.minTemperature = parseFloat(weather.apparent_temperature_min);
 			currentWeather.maxTemperature = parseFloat(weather.apparent_temperature_max);
-			currentWeather.weatherType = this.convertWeatherType(weather.weathercode, currentWeather.isDayTime());
+			currentWeather.weatherType = this.convertWeatherType(weather.weathercode, true);
```

Comment:
how would this show night icons when the isDayTime is now always true?

Reply:
this will never show night icons and this is intended. Using the function `isDayTime` is o.k. for current and hourly but I never want to see night icons in a daily forecast. Did you see the pictures in the issue?

Reply:
Doh!!! Makes perfect sense!